client/src/components/candidate-card.js

```diff
@@ -12,7 +12,11 @@ const CandidateCard = (props) => {
               setUsers(users);
           });
     }, []);
-    let foundUser = users.find(el => el.email === props.user.email);
+    let foundUser;
+    if(props.user){
+      foundUser = users.find(el => el.email === props.user.email);
+    };
+
     const [isSaved, setSaved] = useState(() => {
```

Comment:
const [isSaved, setIsSaved]
-can help keep variable names identifiable.

Reply:
Thank you!